java/code/src/com/redhat/rhn/frontend/xmlrpc/contentmgmt/ContentManagementHandler.java

```diff
@@ -645,6 +651,39 @@ public ContentFilter createFilter(User loggedInUser, String name, String rule, S
         }
     }
 
+    /**
+     * Create new {@link ContentFilter}s for all AppStream modules with default streams
+     *
+     * @param loggedInUser the logged in user
+     * @param prefix the filter name prefix
+     * @param channelLabel label of the modular channel
+     * @param projectLabel label of the Content Lifecycle Project
+     * @throws EntityExistsFaultException when Filter already exist
+     * @return List of created and successfully attached Filter
```

Comment:
The API has additional doc strings to autogenerate the developers documentation.

Reply:
@wombelix we'll add a wiki guide for writing documentation for API methods in Javadoc soon. But until then, please use the existing examples to add a brief documentation to your method.

Basically, you'll need to fill `@xmlrpc.doc` for a description, `@xmlrpc.param` for each parameter with their names and types, and `@xmlrpc.returntype` for the return value.

Reply:
Thanks @mcalmer @cbbayburt, totally forgot about that